src/scratchpads/completion_single_file_fim.rs

```diff
@@ -71,19 +75,47 @@ impl SingleFileFIM {
     }
 }
 
+fn add_context_to_prompt(context_format: &String, prompt: &String, postprocessed_messages: &Vec<ContextFile>) -> String {
+    let mut context_files = vec![];
+    if context_format == "starcoder" {
```

Comment:
Add a default `context_format`
If you cannot figure out which format is the best - just wrap all the code with the comments (see `wrap_comments`) and put it above the prefix part of the code